ko/pl.rst

```diff
@@ -0,0 +1,1383 @@
+
+.. code-block:: sql
+
+    CREATE PROCEDURE print_boolean(b BOOLEAN) AS
+    BEGIN
+        CASE
+            WHEN b THEN
+                put_line('TRUE');
+            WHEN NOT b THEN
+                put_line('FALSE');
+            WHEN b IS NULL THEN
+                put_line('NULL');
+        END;
+    END;
+
+******************
+표현식
+******************
+PL/CUBSQL의 표현식의 종류는 다음 문법으로 요약할 수 있다.
+::
+
+    <expression> ::=
+          <literal>                                 # 상수
+        | <identifier>                              # 식별자
+        | SQL %ROWCOUNT                             # Static SQL 결과 크기
+        | <cursor_expression> <cursor_attribute>    # 커서 속성
+        | <expression> <binary_op> <expression>     # 이항 연산
+        | <unary_op> <expression>                   # 단항 연산
+        | ( <expression> )                          # 괄호
+        | <identifier>.<identifier>                 # 레코드 필드 참조
+        | <identifier> <function_argument>          # 함수 호출
+        | <case_expression>                         # CASE 표현식
+        | <expression> IS [ NOT ] NULL              # IS NULL 표현식
+        | <expression> [ NOT ] BETWEEN <expression> AND <expression>        # BETWEEN 표현식
+        | <expression> [ NOT ] IN <expression> [ , <expression>, ... ]      # IN 표현식
+        | <expression> [ NOT ] LIKE <expression> [ ESCAPE <expression> ]    # LIKE 표현식
+
+    <literal> ::=
+          DATE <quoted_string>
+        | TIME <quoted_string>
+        | (DATETIME | DATETIMETZ | DATETIMELTZ ) <quoted_string>
+        | (TIMESTAMP | TIMESTAMPTZ | TIMESTAMPLTZ ) <quoted_string>
+        | <numeric>
+        | <quoted_string>
+        | { [ <literal> [, <literal> ... ] ] }
+        | NULL
+        | TRUE
+        | FALSE
+    <numeric> ::= UNSIGNED_INTEGER | FLOATING_POINT_NUM
+
+    <cursor_attribute> ::= { %ISOPEN | %FOUND | %NOTFOUND | %ROWCOUNT }
+
+    <binary_op> ::=
+          AND | OR
+        | = | != | <> | ^= | ~= | <= | >= | < | >
+        | * | / | + | - | **
+        | ||
+    <unary_op> ::= + | - | NOT
+
+    <case_expression> ::=
+          CASE <expression> <case_expression_when_part>+ [ ELSE <expression> ] END [ CASE ]
+        | CASE <case_expression_when_part>+ [ ELSE <expression> ] END [ CASE ]
+
+리터럴
+=================
+리터럴에는 날짜/시간(DATE, TIME, DATETIME, TIMESTAMP), 숫자, 문자열, 컬렉션, NULL, TRUE, FALSE 값이 있다.
+비트열을 사용할 수 없다는 점을 제외하고
+`CUBRID SQL 리터럴 <https://www.cubrid.org/manual/ko/11.2/sql/literal.html#>`_\과 동일하다.
+
+식별자
+=================
+Static/Dynamic SQL 밖의 PL/CUBSQL 문에서 사용할 수 있는 식별자에는 다음 세 가지 종류가 있다.
+
+* 선언부에서 선언된 변수, 상수, 커서, Exception, 내부 프로시저/함수
+* 프로시저/함수의 파라메터
+* 암묵적으로 선언된 For 루프의 iterator
+
+명시적 혹은 암묵적 선언 없이 식별자를 사용하면 컴파일 에러가 발생한다.
+
+Static SQL 결과 크기
+====================
+SQL%ROWCOUNT는 Static SQL을 실행한 직후에 결과 크기를 나타내는 표현식이다.
+
+* SELECT 문의 경우 반드시 INTO 절을 포함하게 되고 조회 결과는 1개이어야 한다. 따라서, 이 SELECT 문이 정상적으로 수행되었을 때 SQL%ROWCOUNT의 값은 1이다. 조회 결과 크기가 0이거나 1을 초과해서 실행시간 에러가 발생했을 때에는 SQL%ROWCOUNT의 값은 정의되지 않는다.
```

Comment:
Is this about the SELECT statement not used for cursors?

Reply:
Static SQL's use Implicit Cursors, which is opened and closed automatically. 
The keyword SQL stands for the implicit cursor.

Reply:
SELECT statement using a cursor can have more than one result.
Is this case an exception?

Reply:
SELECT statements associated with a cursor can have multiple results. I'll clarify it in the manual.